src/faebryk/library/DE9Connector.py

```diff
@@ -0,0 +1,38 @@
+# This file is part of the faebryk project
+# SPDX-License-Identifier: MIT
+
+import logging
+
+import faebryk.library._F as F  # noqa: F401
+from faebryk.core.module import Module
+from faebryk.libs.library import L  # noqa: F401
+from faebryk.libs.units import P  # noqa: F401
+
+logger = logging.getLogger(__name__)
+
+
+class DE9Connector(Module):
+    """
+    DE-9 connector
+    """
+
+    # ----------------------------------------
+    #     modules, interfaces, parameters
+    # ----------------------------------------
+    unnamed = L.list_field(9, F.Electrical)
```

Comment:
FLN0005: use pin or terminal not sure whats better for a connector

Reply:
We should call it `contact`. `pin` and `terminal` are already too descriptive